server/methods/permissions/permissionsUpdate.js

```diff
@@ -4,36 +4,27 @@ Meteor.methods({
         function isInArray(value, array) {
 		  return array.indexOf(value) > -1
         }
-		
-        //Get friend and permissions arrays
-        var friendsList = Meteor.call('getPlexFriendlist')
-        var permList = Permissions.find({}, {fields: {_id: 0, permUSER: 1}}).fetch()
-		
-        //Add users to permissions
-        if(permList.length < friendsList.length) {
```

Comment:
I think this was my code from a couple years ago. Oh how times have changed. Glad you caught the bug here but could consider doing comparisons and only updating the difference in the database since the vast majority of the time it will be unchanged for most people. Not anything serious though